packages/fx-core/src/question/create.ts

```diff
@@ -1337,64 +1366,71 @@ export function OfficeAddinFrameworkQuestion(): SingleSelectQuestion {
   };
 }
 
-const officeAddinJsonData = new projectsJsonData();
+/**
+ * when project-type=office-addin-type(office-addin-framework-type=default or react), use selected value;
+ * when project-type=outlook-addin-type, no framework to select, office-addin-framework-type=default_old
+ * when project-type=office-xml-addin-type, no framework to select, office-addin-framework-type=default_old
+ */
+export function getOfficeAddinFramework(inputs: Inputs): string {
+  const projectType = inputs[QuestionNames.ProjectType];
+  if (
+    projectType === ProjectTypeOptions.officeAddin().id &&
+    inputs[QuestionNames.OfficeAddinFramework]
+  ) {
+    return inputs[QuestionNames.OfficeAddinFramework];
+  } else if (
+    projectType === ProjectTypeOptions.officeXMLAddin().id &&
+    inputs[QuestionNames.OfficeAddinHost] !== OfficeAddinHostOptions.outlook().id
```

Comment:
Thanks for the fix in the latest commit.

Please update the related comments in other files. Thanks.